venia-integration-tests/src/fixtures/graphqlMockedCalls/index.js

```diff
@@ -0,0 +1,24 @@
+export const hitGraphqlPath = '**/graphql';
+
+export const getCMSPage = '**/graphql?query=query+GetCmsPage*';
+
+export const getCustomerWishlistCall =
+    '**/graphql?query=query+GetCustomerWishlist*';
+
+export const getMultipleWishlistConfigCall =
+    '**/graphql?query=query+getMultipleWishlistsEnabled*';
+
+export const getWishlistDialogDataCall =
+    '**/graphql?query=query+getWishlistsDialogData*';
+
+export const getWishlistitemsForLocalFieldsCall =
+    '**/graphql?query=query+GetWishlistItemsForLocalField*';
```

Comment:
Slight discrepancy between the `const` and its value here. Is it `LocalField` or `LocalFields`?